### ISSUE TITLE
Preview Image does not display diffusion data from KSampler on the fly during generation via the "Anything Everywhere?" node, but only gets the final image after generation. 

### ISSUE THREAD

**Issue by rollingcookies** [chrisgoringe/cg-controller#294]
If Preview Image is not in a group, it displays the diffusion process from KSampler normally.

**Comment by LukeG89** [comment]
I tested with 3 Preview Image nodes of the same KSampler placed in 3 different places:
- Inside the same group of KSampler
- In a dedicated group (Final Image)
- Outside groups (Ungrouped)

And they are working to me.


https://github.com/user-attachments/assets/b5dc7caf-60c8-46d3-8b21-f8479f67a7b4

**Comment by chrisgoringe** [comment]
@rollingcookies can you give more specifics if you still have the problem?

**Comment by JorgeR81** [comment]
This works for me.

If a `Preview Image` is in a Controller, it also displays the live preview.

@rollingcookies, can you try like this. Just an `Image Preview`, in an single Controller. No groups.

![v1](https://github.com/user-attachments/assets/7589998a-9fb5-4a92-9822-1d0ec1446744)

**Comment by rollingcookies** [comment]
Yes, you are right, forgive me for the false alarm! :/ It was already night last night when testing this feature. 

But the goal was to use a single Preview with on-the-fly diffusion generation data in workflows like this, where there is switching between multiple internal sites that have their own KSampler, so that we can see on-the-fly diffusion generation data in the same window even if we switched to a different KSampler. I started testing the “Any Switch (rgthree)” and your “Anything Everywhere?” nodes for this, and unfortunately yours can't accept data on the fly. 

https://github.com/user-attachments/assets/94e52875-d9e4-4dc0-a88a-265daacb0dd3
